operations/research-and-development/product/development-process/user-interface-text-guidelines.md

```diff
@@ -84,47 +93,113 @@ An error message informs the user that something went wrong. Errors prevent the
+  * **A connection to the Marketplace server could not be established. Check your settings in the **System Console**.**
 
-## Button labels and links
+## In-product copy
 
-Button labels should always use action words and describe the action as concisely as possible. They should be limited to four words or less. Examples: “Log In”, “Send Invitation”.
+The Technical Writers often work with the UI/UX team on in-product copy. The majority of guidelines are available in the [Mattermost Compass Design System](https://zeroheight.com/29be2c109/p/995619-compass-design-system).
 
-## Navigation labels
+### Guidelines for punctuation
 
-Navigation labels should be as short as possible and support the user in finding their way.
+**Headings**
 
-### Input labels
+These are H1 headings such as the title of a modal. Titles shouldn't have periods unless the headline is more than one sentence. Titles also shouldn't contain punctuation such as question marks, colons, semi-colons.
 
-Input labels should be as short and concise as possible and describe the input field.
+**Paragraphs**
 
-### Capitalization
+Paragraphs should always be properly punctuated.
 
-**Titles**
+**Bullets**
 
-Titles should use sentence case \(e.g. **This article is about user interface content**\) with the exception of proper nouns \(e.g. **This article is about Mattermost**\).
+Bullet lists shouldn't have periods unless the bullet text is more than one sentence.
 
-**Button labels, input labels, and navigation labels**
+**Button labels**
 
-These labels should use sentence case. \(e.g. **This article is about user interface content**\) with the exception of proper nouns \(e.g. **This article is about Mattermost**\).
+Button labels shouldn't have periods or other punctuation.
 
-### Punctuation
+### Guidelines for capitalization
 
-**Titles**
+We follow the same capitalization rules across all our documentation assets, including in-product text and UI elements: Always use sentence case (except where you're using a proper noun, of course). This applies to:
 
-Titles shouldn't have periods unless the headline is more than one sentence.
+* Page titles
+* Page headings
+* Section headings
+* Button labels
+* Input labels
+* Navigation labels
+* Menu items
 
-**Paragraphs**
+**Instead of:** This Article is About Mattermost
+**Use:** This article is about Mattermost
 
-Paragraphs should always be properly punctuated.
+### How we write things (WIP)
 
-**Bullets**
+As previously mentioned, we write with empathy in the context of achievement, and in natural language. This can be quite hard to do; as technical writers we generally lean toward very clear and concise writing that can feel clinical. Writing more naturally means we try to avoid convoluted phrasing and we try to make things more simple. 
```

Comment:
```suggestion
As previously mentioned, we write with empathy in the context of achievement using natural language. This can be quite hard to do; as technical writers we generally lean toward very clear and concise writing that can feel clinical. Writing more naturally means we try to avoid convoluted phrasing and we try to make things more simple. 
```